docs/ops.md

```diff
@@ -196,6 +196,8 @@ The only way we can do this is via the config file and not CLI flags.
 In the config file you can specify plugin level options for the set of plugins that you use via the `[plugins.<name>]` sections.
 You will have to read the plugin specific docs to find the options that your plugin accepts.
 
+See [containerd's Plugins documentation](./PLUGINS.md)
```

Comment:
let's make plugin singular s/Plugins/Plugin/
or could remove `containerd's`

Reply:
Made it singular (removed the 's').